### ISSUE TITLE
I want to specify an upstream socks5 proxy service to keep my IP private. 

### ISSUE THREAD

**Issue by TyeGeek** [blokadaorg/blokada#132]

**Comment by TyeGeek** [comment]
Some other adguard apps (like adguard) allow you to specify an upstream proxy to route traffic through after the request makes it through Blokada such that all traffic is routed through the proxy service. This should allow for a username, password, server, and port be specified. Thank you

**Comment by kar** [comment]
Good idea. Have you used it in adguard and did you have any problems with this feature?
